doc/media-info-enhancements/media-info.md

```diff
@@ -0,0 +1,170 @@
+		<td>Validation of port breakout configurations, i.e. whether or not transceiver supports breakout</td>
+	</tr>
+</table>
+<h4>cable_length_detailed</h4>
+<table style="border-style:none;">
+	<tr>
+		<td valign="top">Description</td>
+		<td>Length of directly-attached cable, in metres</td>
+	</tr>
+	<tr>
+		<td valign="top">Value(s)</td>
+		<td>float; 0.0 when no directly-attached cable</td>
+	</tr>
+	<tr>
+		<td valign="top">Needed by</td>
+		<td>Layer-1 configuration lookup, e.g preemphasis</td>
+	</tr>
+	<tr>
+		<td valign="top">Notes</td>
+		<td>Complements existing cable_length field, can represent lengths < 1 m or fractional lengths</td>
+	</tr>
+</table>
+<h4>cable_type</h4>
+<table style="border-style:none;">
+	<tr>
+		<td valign="top">Description</td>
+		<td>Type of directly-attached cable</td>
+	</tr>
+	<tr>
+		<td valign="top">Value(s)</td>
+		<td>string, one of "DAC", "RJ45", "FIBER", "AOC", "ACC"</td>
+	</tr>
+	<tr>
+		<td valign="top">Needed by</td>
+		<td>Layer-1 configuration lookup, e.g NPU interface type</td>
+	</tr>
+</table>
+<h4>display_name</h4>
+<table style="border-style:none;">
+	<tr>
+		<td valign="top">Description</td>
+		<td>String suitable for displaying description of transceiver</td>
+	</tr>
+	<tr>
+		<td valign="top">Value(s)</td>
+		<td>string</td>
+	</tr>
+	<tr>
+		<td valign="top">Needed by</td>
+		<td>CLI show command(s)</td>
+	</tr>
+	<tr>
+		<td valign="top">Notes</td>
+		<td>Rather than the CLI code attempt to construct descriptive string for an installed transceiver, from the information in the TRANSCEIVER_TABLE, xcvrd can construct a short descriptive string for use by CLI show commands.  Since transceiver "knowledge" is inside xcvrd, it is reasonable for it to compose the displayable descriptive string, rather than the CLI.</td>
+	</tr>
+	<tr>
+		<td valign="top">Examples</td>
+		<td>"SFP+ 10GBASE-DWDM-TUNABLE", "QSFP28 100GBASE-CR4-0.5M"</td>
+	</tr>
+</table>
+<h4>lane_count</h4>
+<table style="border-style:none;">
+	<tr>
+		<td valign="top">Description</td>
+		<td>Number of lanes</td>
+	</tr>
+	<tr>
+		<td valign="top">Value(s)</td>
+		<td>integer, 1..N</td>
+	</tr>
+	<tr>
+		<td valign="top">Needed by</td>
+		<td>Validation of port breakout configurations, i.e. whether or not transceiver supports breakout</td>
+	</tr>
+</table>
+<h4>media_interface</h4>
+<table style="border-style:none;">
+	<tr>
+		<td valign="top">Description</td>
+		<td>Media interface</td>
+	</tr>
+	<tr>
+		<td valign="top">Value</td>
+		<td>string, one of "SR", "CR", "ZR", "DR", "EDR", "BASE-T"</td>
+	</tr>
+	<tr>
+		<td valign="top">Needed by</td>
+		<td>Layer-1 configuration lookup, e.g preemphasis</td>
+	</tr>
+</table>
+<h4>form_factor</h4>
+<table style="border-style:none;">
+	<tr>
+		<td valign="top">Description</td>
+		<td>Detailed transceiver type, i.e. form factor</td>
+	</tr>
+	<tr>
+		<td valign="top">Value(s)</td>
+		<td>string, one of "SFP", "SFP+", "SFP28", "QSFP+", "QSFP28", "QSFP28-DD", "QSFP56-DD", "RJ45"</td>
```

Comment:
missing osfp, sfp56

Reply:
there is a type_abbrv_name, what is the augment here?

Reply:
This list is for the first version -- it is not intended to be complete.  As new media form factors are created, or supported, new data values will be added.